src/ReflectionParameter.php

```diff
@@ -24,6 +29,11 @@ class ReflectionParameter implements \Reflector
     private $isOptional;
 
     /**
+     * @var bool
+     */
+    private $hasDefaultValue;
```

Comment:
Why? You can't instantiate the class without setting it to a `bool`, so it will never be anything but a `bool`...?